pkg/cli/lambda_command.go

```
@@ -0,0 +1,391 @@
+}
+
+func (l *staticLambdaResolver) ResolveEndpoint(ctx context.Context, params lambda_sdk.EndpointParameters) (aws_transport.Endpoint, error) {
+	return aws_transport.Endpoint{
+		URI: *l.endpoint,
+	}, nil
+}
+
+func newStaticLambdaResolver(endpoint string) (lambda_sdk.EndpointResolverV2, error) {
+	uri, err := url.Parse(endpoint)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse endpoint: %v", err)
```

golangci-lint (1.62.2)

non-wrapping format verb for fmt.Errorf. Use `%w` to format errors

(errorlint)